src/RubberDuckyParking/parking_api/migrations/0001_initial.py

```diff
@@ -143,6 +143,16 @@ class Migration(migrations.Migration):
                 'verbose_name_plural': 'Transactions',
             },
         ),
+        migrations.CreateModel(
+            name='Reservation',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('start_date', models.DateTimeField()),
+                ('end_date', models.DateTimeField()),
```

Comment:
Should there be a time that the reservation was purchased/ made?

Reply:
we can add that nbd, I can make a migration for it really quick.